conbench/__init__.py

```diff
@@ -1,7 +1,10 @@
 import importlib.metadata as importlib_metadata
```

Comment:
You mentioned

>I override that to DEBUG in my local test environment.

Would you mind adding a quick example to the README, in the "run tests" section, about how you do that? It's probably a good practice for other devs as well!

Reply:
Great idea! 

I will keep observing my dev flow for another while though before I feel comfortable updating the README -- right now I run tests with docker-compose, and I do the logging level change by adding
```
      CONBENCH_LOG_LEVEL_STDERR: DEBUG
```
to the app/environment section in  my local checkout of `docker-compose.yml`.